Cargo.toml

```diff
@@ -28,7 +28,7 @@ path = "tests/tests.rs"
 [dependencies]
 atty = "0.2.2"
 bytecount = "0.1.4"
-clap = "2.20.5"
```

Comment:
I'm not quite certain that this should be bumped (similarly for `build-dependencies`). A `cargo update` should be sufficient.